lib/datasource/artifactory/__snapshots__/index.spec.ts.snap

```diff
@@ -5,19 +5,19 @@ Object {
   "registryUrl": "https://jfrog.company.com/artifactory",
   "releases": Array [
     Object {
-      "releaseTimestamp": "2021-07-21T20:08:00.000Z",
+      "releaseTimestamp": "2021-07-22T00:08:00.000Z",
```

Comment:
These changes looks wrong to me.

Reply:
That was the result of `npm run jest -- -u`.  I'm happy to revert changes to this file

Reply:
Ideally we don't have any timestamps, but if we do then they don't change. Will people running `jest` locally end up with this test failing?

Reply:
Those shouldn't changes, as they are coming from fixtures. Looks like we have a timezone issue in artifactory

Reply:
I was able to recreate the above snapshot timestamp discrepancy on `main`

Running `npm run test` works as committed. 

Running `npm run jest` produces an update to the artifactory snapshot.

In either case, I have reverted this change and only updated the bitbucket snapshot associated with my changes